Scripts/Core/VRView.cs

```diff
@@ -132,9 +133,8 @@ public void OnEnable()
 			m_Camera.farClipPlane = 1000f;
 
 			// Generally, we want to be at a standing height, so default to that
-			const float kHeadHeight = 1.7f;
 			Vector3 position = m_CameraRig.position;
-			position.y = kHeadHeight;
+			position.y = HeadHeight;
```

Comment:
We should also check if the tracking is configured for sitting or standing and not apply the offset if it is standing/roomscale. Since I am just exposing this for the reset feature, I would like to punt this to a later improvement. I can create an issue if needed.